src/main/java/nl/knaw/dans/migration/core/ExpectedLoader.java

```diff
@@ -79,15 +82,23 @@ public void retriedSave(ExpectedFile expected) {
     }
   }
 
+  @UnitOfWork("hibernate")
   public void saveExpectedFile(ExpectedFile expected) {
-      log.trace(expected.toString());
       expectedFileDAO.create(expected);
   }
 
+  @UnitOfWork("hibernate")
   public void saveExpectedDataset(ExpectedDataset expected) {
-      String depositor = expected.getDepositor();
-      expected.setDepositor(accountSubStitues.getOrDefault(depositor, depositor));
-      log.trace(expected.toString());
-      expectedDatasetDAO.create(expected);
+    String depositor = expected.getDepositor();
+    expected.setDepositor(userToEmail.getOrDefault(depositor, depositor));
+
+    if (null != expected.getLicenseUrl()) {
+      String url = expected.getLicenseUrl()
+              .replace("https://", "http://")
+              .replaceAll("(.html|.txt|.pdf)?/$", "");
+      expected.setLicenseName(licensesUrlToName.getOrDefault(url, null));
```

Comment:
@lindareijnhoudt: Please verify these replacements cover all variations found in real data.